### PR TITLE
Remote platform documentation for AndroidTV

### PR DESCRIPTION
**Pull request by ollo69**
## Type of change

Remote platform documentation for AndroidTV integration

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/103496
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Added support for the `remote` platform, enabling users to send commands to Android/Fire TV devices using the `remote.send_command` service.
  - Provided a detailed list of supported key commands and their descriptions.
  - Included examples for sending command sequences.

### CODE REVIEW

File: source/_integrations/androidtv.markdown [hunk 1]
```diff
@@ -2,7 +2,8 @@
 title: Android Debug Bridge
 description: Instructions on how to integrate Android and Fire TV devices into Home Assistant.
 ha_category:
-  - Media player
+  - Media Player
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown]:
```suggestion
  - Media player
```

File: source/_integrations/androidtv.markdown [hunk 2]
```diff
@@ -243,3 +245,99 @@ The solution to this problem is the `state_detection_rules` configuration parame
...
+"MOVE_HOME"|Move Home
+"PAIRING"|Pairing
+"REWIND"|Rewind
+"RIGHT"|Right
+"SEARCH"|Search
+"SETTINGS"|Settings
+"SYSDOWN"|Sysdown
+"SYSLEFT"|Sysleft
+"SYSRIGHT"|Sysright
+"SYSUP"|Sysup
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown:320]: What does Sys mean? How is Sysdown different from down?

File: source/_integrations/androidtv.markdown [hunk 3]
```diff
@@ -243,3 +245,99 @@ The solution to this problem is the `state_detection_rules` configuration parame
...
+**Other Keys**
+Key|Description
+---|-----------
+"BACK"|Back
+"CENTER"|Center
+"DOWN"|Down
+"END"|End
+"ENTER"|Enter
+"ESCAPE"|Escape
+"FAST_FORWARD"|Fast Forward
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown]:
```suggestion
"FAST_FORWARD"|Fast forward
```

File: source/_integrations/androidtv.markdown [hunk 4]
```diff
@@ -243,3 +245,99 @@ The solution to this problem is the `state_detection_rules` configuration parame
...
+"HOME"|Home
+"LEFT"|Left
+"MENU"|Menu
+"MOVE_HOME"|Move Home
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown]:
```suggestion
"MOVE_HOME"|Move home
```

File: source/_integrations/androidtv.markdown [hunk 5]
```diff
@@ -243,3 +245,99 @@ The solution to this problem is the `state_detection_rules` configuration parame
...
+{% details "Full keycodes list" %}
+
+**Power Keys**
+Key|Description
+---|-----------
+"POWER"|Power Toggle
+"SLEEP"|Sleep Mode
+"RESUME"|Resume
+"SUSPEND"|Suspend Mode
+"WAKEUP"|Wake Up
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown]:
```suggestion
"POWER"|Power toggle
"SLEEP"|Sleep mode
"RESUME"|Resume
"SUSPEND"|Suspend mode
"WAKEUP"|Wake up
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization)

File: source/_integrations/androidtv.markdown [hunk 6]
```diff
@@ -243,3 +245,99 @@ The solution to this problem is the `state_detection_rules` configuration parame
...
+____________
+
+**Input Keys**
+Key|Description
+---|-----------
+"COMPONENT1"|Component1
+"COMPONENT2"|Component2
+"COMPOSITE1"|Composite1
+"COMPOSITE2"|Composite2
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown]:
```suggestion
"COMPONENT1"|Component 1
"COMPONENT2"|Component 2
"COMPOSITE1"|Composite 1
"COMPOSITE2"|Composite 2
```

File: source/_integrations/androidtv.markdown [hunk 7]
```diff
@@ -243,3 +245,99 @@ The solution to this problem is the `state_detection_rules` configuration parame
...
+"HDMI1"|HDMI1
+"HDMI2"|HDMI2
+"HDMI3"|HDMI3
+"HDMI4"|HDMI4
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown]:
```suggestion
"HDMI1"|HDMI output port 1
"HDMI2"|HDMI output port 2
"HDMI3"|HDMI output port 3
"HDMI4"|HDMI output port 4
```
I'm not sure, but I'm assuming these are output ports? Please correct me if I got this wrong.

File: source/_integrations/androidtv.markdown [hunk 8]
```diff
@@ -243,3 +245,99 @@ The solution to this problem is the `state_detection_rules` configuration parame
...
+"INPUT"|Change Input
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown]:
```suggestion
"INPUT"|Change input
```

File: source/_integrations/androidtv.markdown [hunk 9]
```diff
@@ -243,3 +245,99 @@ The solution to this problem is the `state_detection_rules` configuration parame
...
+"SAT"|Sat
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown]:
```suggestion
"SAT"|Satellite
```

File: source/_integrations/androidtv.markdown [hunk 10]
```diff
@@ -243,3 +245,99 @@ The solution to this problem is the `state_detection_rules` configuration parame
...
+"VGA"|VGA
+_____________
+
+**Volume Keys**
+Key|Description
+---|-----------
+"VOLUME_DOWN"|Volume Down
+"VOLUME_UP"|Volume Up
+"MUTE"|Volume Mute
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown]:
```suggestion
"VOLUME_DOWN"|Volume down
"VOLUME_UP"|Volume up
"MUTE"|Volume mute
```

File: source/_integrations/androidtv.markdown [hunk 11]
```diff
@@ -243,3 +245,99 @@ The solution to this problem is the `state_detection_rules` configuration parame
...
+"SYSLEFT"|Sysleft
+"SYSRIGHT"|Sysright
+"SYSUP"|Sysup
+"TEXT"|Text
+"TOP"|Top
+"UP"|Up
+
+{% enddetails %}
+
+You can also send others Android keys using the syntax `input keyevent {key}`, replacing `{key}` with the Android numeric key event. Refer to [Android TV KeyEvent](https://developer.android.com/reference/android/view/KeyEvent) for details.
```
Comment (c0ffeeca7) [inline source/_integrations/androidtv.markdown]:
```suggestion
You can also send other Android keys using the syntax `input keyevent {key}`, replacing `{key}` with the Android numeric key event. Refer to [Android TV KeyEvent](https://developer.android.com/reference/android/view/KeyEvent) for details.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Tiny tweaks

File: source/_integrations/androidtv.markdown (hunk 2 above)
Comment (ollo69) [reply source/_integrations/androidtv.markdown:320]: Good question, honestly I don't know. I take the values from the integration library [here](https://github.com/JeffLIrion/python-androidtv/blob/d8a8f3a4810eec7180f0370750ea6d216dd8300f/androidtv/constants.py#L352). If you think that does not make sense we can remove.

Comment (c0ffeeca7) [reply source/_integrations/androidtv.markdown:320]: No, I was just hoping for the description to be more descriptive.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @ollo69 :+1:

File: source/_integrations/androidtv.markdown (hunk 2 above)
Comment (c0ffeeca7) [reply source/_integrations/androidtv.markdown:320]: My comment is a bit late, but maybe you could create a follow-up PR:
<https://learn.microsoft.com/en-us/dotnet/api/android.views.keycode?view=net-android-34.0>
They have a bit more descriptive docs.
looked up code 19 for UP: `19	
Key code constant: Directional Pad Up key. May also be synthesized from trackball motions. `
and 280 for `SystemNavigationUp`.

**Pull request merged**